src/style-auro-calendar.scss

```diff
@@ -6,38 +6,39 @@
 @import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
 @import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
 
-/* stylelint-disable color-function-notation, at-rule-empty-line-before */
+/* stylelint-disable color-function-notation, at-rule-empty-line-before, order/properties-order */
 
 :host {
-  --mobile-header-height: 86px;
-  --mobile-footer-height: 200px;
-  --calendar-width: 336px;
-
   @include auro_breakpoint($max: $auro-breakpoint-sm) {
     flex-direction: column;
   }
+
+  --calendar-width: 336px;
+  --mobile-footer-height: 200px;
+  --mobile-header-height: 86px;
 }
 
 .calendarNavBtn {
   display: flex;
   width: var(--auro-size-500);
   height: var(--auro-size-500);
+
   align-items: center;
   justify-content: center;
+
   border-width: 1px;
   border-style: solid;
   border-color: transparent;
   border-radius: 50%;
+
   background-color: transparent;
   color: var(--auro-color-border-active-on-light);
-  cursor: pointer;
 
-  &:hover {
-    border-color: var(--auro-color-border-active-on-light);
-  }
+  cursor: pointer;
 
   auro-icon {
```

Comment:
There are still a lot of references to `auro-icon` that are not null that we are not using that element. 

```
style-auro-calendar.scss
39:  auro-icon {

auro-calendar.js
339:            <auro-icon category="interface" name="chevron-left" customColor></auro-icon>

style-auro-calendar-month.scss
75:  auro-icon {
```